ui-modules/blueprint-composer/app/components/spec-editor/spec-editor.directive.js

```diff
@@ -406,11 +414,18 @@ export function specEditorDirective($rootScope, $templateCache, $injector, $sani
             }
             return $filter('specEditorConfig')(allConfig, scope.state.config.filter.values).indexOf(config) === -1;
         };
+        specEditor.recordFocus = (subsection, name) => {
+            scope.state.focus.subsection = subsection;
+            scope.state.focus.name = name;
+        };
```

Comment:
A single subsection must be in focus only. Previously, only the last subsection was focused on refresh (when DSL menu closes) if multiple expected to be in focus, like property or configuration at the same time.